ivo-projects/todo/src/components/App.js

```diff
@@ -93,13 +62,14 @@ class App extends Component {
                             onClick={(e) => this.addReminder(e)}
                             disabled={this.state.disabled}
                         >
-                            Add this
+                        Add todo
                         </button>
                     </div>
                     <RemindersList
                         reminders={this.props.reminders}
                         deleteReminder={this.props.deleteReminder}
-                        updateInterval="1000"
+                        editItem={this.state.editItem}
```

Comment:
What is editItem used for?

Reply:
this was not meant to be commited...I must find a way to compare my lines of code before commiting..not only seeing the modified files in git bash..

Reply:
Can you remove it?